src/openforms/appointments/templates/appointments/appointment__pdf.html

```diff
@@ -1,3 +1,4 @@
+{% autoescape off %}
```

Comment:
I assume this is fine, as it is only being rendered in a PDF

Reply:
As discussed during standup: this could potentially be unsafe even in PDF files. Perhaps it makes sense to escape (the default with the DTL) the rendered variables after all. I don't expect the values coming from the appointment plugins to be HTML anyway. If necessary (e.g. for the QR code), `format_html` is called

Reply:
And I don't know if any user input are rendered in the appointment PDF, but if it is the case then it makes sense to actually escape it

Reply:
The product names, location names etc. that we retrieve from external services are *also* untrusted user input, so they need to be escaped indeed.